scripts/install.sh

```
@@ -626,13 +621,119 @@ EOF
-  echo "try the .deb package from https://github.com/${REPO}/releases/latest"
-  echo "(Debian/Ubuntu) or report at https://github.com/${REPO}/issues with your"
+  echo "If the AppImage prints 'Interpreter not found!', unshare/uid_map errors,"
+  echo "or missing system-library errors such as libgbm.so.1, use the .deb package"
+  echo "from https://github.com/${REPO}/releases/latest on Debian/Ubuntu."
+  echo "For other distros, report at https://github.com/${REPO}/issues with your"
   echo "distro, kernel (uname -a), GPU driver (lspci), dmesg excerpt, and asset: ${ASSET_NAME}."
   echo "Uninstall: rm -f \"${app_path}\" \"${desktop_file}\""
 }
 
+if [[ "${SOURCE_ONLY}" == "1" ]]; then
+  return 0 2>/dev/null || exit 0
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)